aws/auto-scaling-group/vpc.tf

```diff
@@ -101,6 +101,16 @@ resource "aws_vpc_security_group_ingress_rule" "allow_ssh_ingress" {
   to_port           = 22
 }
 
+resource "aws_vpc_security_group_ingress_rule" "allow_custom_ingress" {
+  count = length(var.target_groups)
+
+  security_group_id = aws_security_group.sg.id
+  cidr_ipv4         = "0.0.0.0/0"
+  from_port         = var.target_groups[count.index].port
+  ip_protocol       = "tcp"
+  to_port           = var.target_groups[count.index].port
+}
+
```

Comment:
I don't think we need egress rules for target group ports by default. The existing ones are there so that the instance can access webpages/registries/etc during initial setup. We probably need another default egress rule for port 443 but that is out of scope of this PR